src/trans/gpu/algor/buffered_allocator_mod.F90

```diff
@@ -121,18 +121,20 @@ FUNCTION GET_ALLOCATION(ALLOCATOR, RESERVATION)
   END FUNCTION GET_ALLOCATION
 
   SUBROUTINE ASSIGN_PTR_FLOAT(DST, SRC, START_IN_BYTES, LENGTH_IN_BYTES, SET_VALUE, SET_STREAM)
-    USE ISO_C_BINDING, ONLY: C_FLOAT, C_F_POINTER, C_SIZEOF
+    USE ISO_C_BINDING, ONLY: C_FLOAT, C_F_POINTER, C_SIZEOF, C_SIZE_T
     IMPLICIT NONE
     INTEGER(KIND=C_INT8_T), POINTER, INTENT(IN) :: SRC(:)
     REAL(KIND=C_FLOAT), POINTER, INTENT(OUT) :: DST(:)
     LOGICAL, INTENT(IN), OPTIONAL :: SET_VALUE
     INTEGER(KIND=4), INTENT(IN), OPTIONAL :: SET_STREAM
     LOGICAL :: SET_VALUE_EFF
     INTEGER(KIND=4) :: SET_STREAM_EFF
-    INTEGER(KIND=C_SIZE_T) :: START_IN_BYTES, LENGTH_IN_BYTES, END_IN_BYTES, J
-    IF (START_IN_BYTES + LENGTH_IN_BYTES - 1 > SIZE(SRC, KIND=C_SIZE_T)) THEN
+    INTEGER(KIND=JPIB) , INTENT(IN) :: START_IN_BYTES, LENGTH_IN_BYTES
+    INTEGER(KIND=JPIB) :: END_IN_BYTES, J
+    END_IN_BYTES=START_IN_BYTES + LENGTH_IN_BYTES - 1
+    IF (END_IN_BYTES > SIZE(SRC, KIND=C_SIZE_T)) THEN
       CALL ABORT_TRANS("Logical Error in ASSIGN_PTR - OOB assignment")
-    ENDIF
+   ENDIF
```

Comment:
```suggestion
    ENDIF
```
Indentation typo.

Reply:
Still there. BTW Github's "add suggestion" buttons there make it super easy to fix these annoying tiny changes.